lib/datasource/remote/polkadot_api/polkadot_repository.dart

```diff
@@ -301,11 +301,12 @@ class PolkadotRepository extends KeyRepository {
         res['address'] = address;
         guardiansModel = GuardiansConfigModel.fromJson(res);
       } else {
-        guardiansModel = GuardiansConfigModel.empty;
+        return Result.error(ErrorResult("Error Loading Guardians"));
       }
-      return Future.value(Result.value(guardiansModel));
```

Comment:
future not needed

Reply:
this returns error or model.

Reply:
In my mind there were 3 cases:
- Guardians load
- Guardians load but user hasn't set any - I am not entirely sure what that returns from the polkadot call. Need to test it. 
- Load error (of any kind)